src/screens/Analytics.tsx

```diff
@@ -0,0 +1,12 @@
+import { Container } from "@mui/material";
+import ComposedTwoAreasChart from "../component/ComposedTwoAreasChart";
+import ComposedTwoYAxisChart from "../component/ComposedTwoYAxisChart";
+
+export function Analytics() {
```

Comment:
We should either default export everywhere or use named exports everywhere. Actually I have no preference between the two as long as it's consistent throughout the app.